Registry returned 404 for GET on https://registry.npmjs.org/esprima-six

I can't run `npm start` successfully because of the following error:

`npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/esprima-six
npm ERR! 404 
npm ERR! 404 'esprima-six' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'derequire'`

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "Chrome extension to quickly switch to other open tabs with just your keyboard",
   "devDependencies": {
     "watchify": "~0.6.1",
-    "browserify": "~3.30.0",
+    "browserify": "~13.0.0",
     "nodeunit": "~0.8.5",
     "q": "~1.0.0",
     "jshint": "~2.4.3",

